code/game/objects/obj_defense.dm

```diff
@@ -229,12 +229,10 @@ GLOBAL_DATUM_INIT(acid_overlay, /mutable_appearance, mutable_appearance('icons/e
 ///Called when the obj is hit by a tesla bolt.
 /obj/proc/zap_act(power, zap_flags, shocked_targets)
 	if(QDELETED(src))
-		return
+		return 0
```

Comment:
nothing checks return value

Reply:
All of the power assignments check the return value, so I put the 0 there as I'm not sure what the default return type is, and wanted to be sure.

Reply:
nothing that uses zap_act() that i can see checks the return value of it, even parent calls of zap_act() dont check parent return value

Reply:
Unless I am doing this incorrectly these should be using the return value.
![image](https://user-images.githubusercontent.com/58055496/74598698-e6d83900-502a-11ea-941f-97ddd89d6d67.png)

Reply:
oh i missed those.